### PR TITLE
adding ethics-committee-with-text image

### LINKED ISSUE #3345: [Image needs adding] Research Ethics Committees Chapter
### Summary

The chapter: https://the-turing-way.netlify.app/ethical-research/ethics-committees does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

### What needs to be done?

- [x] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [x] Write the alt text for the image
- [x] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by CeilidhWelsh**
Closes #3345

### CODE REVIEW

File: book/website/ethical-research/ethics-committees.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ This is because they can provide ethical oversight to expert researchers who do
 This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work -- but to provide support in respect to these questions.
 Whilst there is a great deal of complexity involved in holding individuals accountable in scientific projects, this chapter assumes that RECs seek to improve how science is conducted at large.
 
+```{figure} ../../figures/ethics-committee-with-text.*
+---
+height: 500px
+name: ethics-committee-with-text
+alt: A female presenting person is walking into a path that leads to somewhere far with mountains. The path starts with a big banner that says “Research Ethics Committee”. At the beginning of the path, there is a male presenting person with a map, a backpack and a friendly face looking like they are ready to join the other person to embark on the research ethics committee journey. Past the first part of the path, there are a couple of booths, one with “Ethics Reviewers”, and another one further down with “ethics decision making” as the title of the booth. These represent the journey one goes through when getting into the research ethics committee processes.
```
Comment (aleesteele) [inline book/website/ethical-research/ethics-committees.md:16]:
```suggestion
alt: A female-presenting person is walking into a path that leads to somewhere far with mountains. The path starts with a big banner that says “Research Ethics Committee”. At the beginning of the path, there is a male-presenting person with a map, a backpack, glasses and a friendly face looking like they are ready to join the other person to embark on their research ethics committee journey. Next to the first part of the path, there are a couple of booths full of people, one with “Ethics Reviewers”, and another one further down with “Ethics Decision Making” as the title of the booth. These represent the journey one goes through when going through research ethics committee processes.
```

**Comment by aleesteele** [comment]
Just added some comments, but it looks like the build is failing. Thanks @CeilidhWelsh and @Susana465!

File: book/website/ethical-research/ethics-committees.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ This is because they can provide ethical oversight to expert researchers who do
 This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work -- but to provide support in respect to these questions.
 Whilst there is a great deal of complexity involved in holding individuals accountable in scientific projects, this chapter assumes that RECs seek to improve how science is conducted at large.
 
+```{figure} ../../figures/ethics-committee-with-text.*
```
Comment (EKaroune) [inline book/website/ethical-research/ethics-committees.md:12]:
```suggestion
```{figure}../../figures/ethics-committee-with-text.*
```

**Review by EKaroune — APPROVED** [review]
check that the image does render in the preview and then it will be ready to go 🚀

**Comment by Arielle-Bennett** [comment]
This build is failing - error: 

`/Users/runner/work/the-turing-way/the-turing-way/book/website/ethical-research/ethics-committees.md:12: WARNING: Pygments lexer name '{figure}../../figures/ethics-committee-with-text.*' is not known`

File: book/website/ethical-research/ethics-committees.md [hunk 3]
```diff
@@ -9,6 +9,15 @@ This is because they can provide ethical oversight to expert researchers who do
 This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work -- but to provide support in respect to these questions.
 Whilst there is a great deal of complexity involved in holding individuals accountable in scientific projects, this chapter assumes that RECs seek to improve how science is conducted at large.
 
+```{figure}../../figures/ethics-committee-with-text.*
```
Comment (EKaroune) [inline book/website/ethical-research/ethics-committees.md:12]:
```suggestion
```{figure} ../../figures/ethics-committee-with-text.*
```

File: book/website/ethical-research/ethics-committees.md (hunk 2 above)
Comment (EKaroune) [inline book/website/ethical-research/ethics-committees.md:12]:
```suggestion
```{figure} ../figures/ethics-committee-with-text.*
```

**Comment by EKaroune** [comment]
@all-contributors please add @CeilidhWelsh for content

**Comment by EKaroune** [comment]
@all-contributors please add @Susana465 for content

**Comment by EKaroune** [comment]
https://github.com/all-contributors please add @Susana465 for content

**Comment by kallewesterling** [comment]
Closing this branch, and moving issue with adding @Susana465 as a contributor to a separate issue. See #3399.

**Pull request merged**